scanrepository/scanrepository.go

```diff
@@ -122,9 +122,20 @@ func (cfp *ScanRepositoryCmd) scanAndFixBranch(repository *utils.Repository) (er
 }
 
 func (cfp *ScanRepositoryCmd) setCommandPrerequisites(repository *utils.Repository, client vcsclient.VcsClient) (err error) {
+	repositoryInfo, err := client.GetRepositoryInfo(context.Background(), cfp.scanDetails.RepoOwner, cfp.scanDetails.RepoName)
```

Comment:
Apply the change I sent you. This causes a nil pointer since we address scanDetails before creating them.
We need to create scanDetails -> get repositoryInfo -> perform the new additions you added in createXrayScanParams